cocos/core/assets/simple-texture.ts

```diff
@@ -226,8 +226,11 @@ export class SimpleTexture extends TextureBase {
         if (!device) {
             return;
         }
+        // create a new texture view before the destruction of the previous one to bypass the bug that
+        // vulkan destroys textureview in use. This is a temporary solution, should be fixed later.
+        const textureView = this._createTextureView(device);
```

Comment:
Since_createTextureView now can be null, public getGFXTexture () will return null.
Is this correct behabior?